<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JrdGui</class>
 <widget class="QMainWindow" name="JrdGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JrdGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabGame">
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>481</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupGameControl">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>0</y>
          <width>141</width>
          <height>321</height>
         </rect>
        </property>
        <property name="title">
         <string>Game Control</string>
        </property>
        <widget class="QPushButton" name="buttonStart">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>141</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Start game</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>110</y>
           <width>113</width>
           <height>194</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buttonEnablePlayer">
            <property name="text">
             <string>enablePlayer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDisablePlayer">
            <property name="text">
             <string>DisablePlayer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRunPlayer">
            <property name="text">
             <string>RunPlayer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonStopPlayer">
            <property name="text">
             <string>StopPlayer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMode1Player">
            <property name="text">
             <string>Mode1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMode3Player">
            <property name="text">
             <string>Mode3</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tabPatient">
       <attribute name="title">
        <string>Patient</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabProtocol">
       <attribute name="title">
        <string>Protocol</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabSetROM">
       <attribute name="title">
        <string>SetROM</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabReport">
       <attribute name="title">
        <string>Report</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView_2">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuView_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
